sample-tracking-status-overview-app/src/main/groovy/life/qbic/portal/sampletracking/components/projectoverview/ProjectOverviewView.groovy

```diff
@@ -194,15 +194,8 @@ class ProjectOverviewView extends VerticalLayout{
         viewModel.generatedManifest = null
     }
 
-    private static String getStyleForColumn(ProjectSummary projectSummary, ValueProvider<ProjectSummary, RelativeCount> valueProvider) {
-        RelativeCount relativeCount = valueProvider.apply(projectSummary)
-        State state = determineCompleteness(relativeCount)
-        return state.getCssClass()
-    }
-
-    private static String getStyleForFailureColumn(ProjectSummary projectSummary, ValueProvider<ProjectSummary, RelativeCount> valueProvider) {
-        RelativeCount relativeCount = valueProvider.apply(projectSummary)
-        State state = determineFailure(relativeCount)
+    private static String getStyleForColumn(ProjectSummary projectSummary, SampleCount sampleStatusCount) {
```

Comment:
I am very pleased by this method :+1: Makes me happy that you found a way to get rid of the special behaviour of one status 🚀